destination/destination_integration_test.go

```diff
@@ -258,9 +258,6 @@ func TestDestination_Write_successKeyColumns(t *testing.T) {
 	_, err = db.Exec(fmt.Sprintf("INSERT INTO %s VALUES (1, 2);", cfg[config.ConfigTable]))
 	is.NoErr(err)
 
-	// set a KeyColumns field to the config
-	cfg[config.ConfigKeyColumns] = "col1"
```

Comment:
We can keep it also, to accept the keys from config now that I know [this](https://github.com/conduitio-labs/conduit-connector-redshift/pull/97#discussion_r1804439177)
Since it accepted multiple keys for table and we had to take multiple tables from config, so I removed it as it was not a required field and we could fetch the keys ourselves also.

Anyways I will take this from config like
tables.tableNameFoo.keyColumns:  columnFoo,columnFoo1
tables.tableNameBar.keyColumns:  columnBar